Resources/Prototypes/Entities/Clothing/Masks/bandanas.yml

```diff
@@ -68,7 +69,6 @@
     sprite: Clothing/Head/Bandanas/botany.rsi
   - type: Tag
     tags:
-    - ClothMade
```

Comment:
Keep this line. Tags are overwritten, not merged, so without it the botany bandana would not be edible.

Reply:
Oh, gotcha. So it'd either be remove the tags entirely from the item or keep the ClothMade tag? Will do.